python/cugraph/cugraph/structure/graph_implementation/simpleDistributedGraph.py

```diff
@@ -64,8 +64,6 @@ def __from_edgelist(
     ):
         if not isinstance(input_ddf, dask_cudf.DataFrame):
             raise TypeError("input should be a dask_cudf dataFrame")
-        if self.properties.directed is False:
-            raise TypeError("Undirected distributed graph not supported")
```

Comment:
Since this removes the restriction of directed-only graphs for all of our MG algos, we'll need to make sure all of our tests use both directed and undirected graphs as inputs.  For instance, MG louvain is only tested with directed ([here](https://github.com/rapidsai/cugraph/blob/eae7795ee3ff0fc7df9f460cfc389a0d5b9606be/python/cugraph/cugraph/tests/dask/test_mg_louvain.py#L67)).  We should parameterize our MG tests and ensure they either pass for both graph types, or raise the appropriate exception (`ValueError`) when passed an undirected if that isn't supported.

Reply:
Understood Working on this now.